### ISSUE TITLE
Hard to find somewhat basic options.

### ISSUE THREAD

**Issue by Transisto** [BlueWallet/BlueWallet#2989]
Where can we instruct the wallet to allow spending of unconfirmed funds?

Stuck with a very low fee incoming transaction and can't bump it via CPFP, even after enabling advanced option I can't find it anywhere.

RBF??? 
How can we chose whether we want a transaction to be RBF or not? Some service will only consider non-rbf for zero-conf. 

I'm not sure if Blue even support RBF, I've read somewhere that only BIP84 wallets have the RBF function, and I don't see how that's related.

**Comment by ncoelho** [comment]
Yes, RBf and CPFP are available on bech32 wallets. RBF is on by default on the send > options tab.

https://bluewallet.io/high-fees-and-transactions-pending-what-to-do/

**Comment by Transisto** [comment]
Isn't it harder limit these feature to Bech32? while any address types supports it?

**Comment by ncoelho** [comment]
It isn't limited, it was just not done to all types, which can be a lot and a lot of work.

**Comment by Transisto** [comment]
I think it's safe to assume that a lot of mid-advanced users will be looking around a lot for these features when using the more compatible BIP39 addresses.

**Comment by Transisto** [comment]
Does it also mean that "Spend unconfirmed" is only available to BECH32 addresses?

FYI, about 80% of retail customers still use blockchain info as their wallet. Which isn't compatible with bc1 addresses.
Would be great to be able to use RBF + CPFP at least with Segwit compatible addresses.

**Comment by ncoelho** [comment]
I believe it does support it.

I'm just informing about the support, no one is limiting or deciding to not have that feature. No one has done it yet, this is an open source project.
